Delicate_Flowers_Vignette-AR.qmd

```diff
@@ -0,0 +1,222 @@
+
+[^delicate_flowers_vignette-ar-4]: [Emma LeConte, “Diary, 1864-1865,” (Accessed February 12, 2024](https://www.zotero.org/google-docs/?broken=0ZgaEi)), 13.
+
+[^delicate_flowers_vignette-ar-5]: Margaret Ann Meta Morris Grimball, “Journal of Meta Morris Grimball, South Carolina, December 1860-February 1866: Electronic Version,” (Documenting the South: UNC, 1998), 41.
+
+Each of the topic models below was created using seven Confederate women’s diaries. The first looks at women who were 21 years old or younger when the war started, and the second looks at women who were 26 or older. Although Southern women often married young, in my corpus those under 21 years of age were unmarried. There are more topics in the first model because the young women in this particular sample wrote more than their older counterparts. 
+
+```{r}
+diary_young <- diary_whole_unnest %>% filter(name == "Sarah Morgan Dawson" | name == "Eliza Frances Andrews" | name == "Belle Edmondson" | name == "Emma LeConte" | name == "Sarah Lois Wadley" | name == "Kate S. Carney" | name == "Susan R. Jervey")
+
+diary.dtm <- diary_young %>% 
+  count(file_name, word) %>% 
+  cast_dtm(file_name, word, n)
+
+diary.lda <- LDA(diary.dtm, k = 13, control = list(seed = 12345),  method = "Gibbs", alpha = .5)
+diary.lda
+
+diary.topics <- tidy(diary.lda, matrix = "beta")
+head(diary.topics)
+
+diary.top.terms <- diary.topics %>%
+  arrange(desc(beta)) %>% 
+  group_by(topic) %>% slice(1:8)
+
+
+diary.top.terms %>%
+  mutate(term = reorder_within(term, beta, topic)) %>%
+  ggplot(aes(beta, term, fill = factor(topic))) +
+  geom_col(show.legend = FALSE) +
+  facet_wrap(~ topic, scales = "free") +
+  scale_y_reordered()
+```
+
+```{r}
+diary_old <- diary_whole_unnest %>% filter(name == "Dolly Sumner Lunt" | name == "Anita Dwyer Withers" | name == "Mary Boykin Chesnut" | name == "Meta Morris Grimball" | name == "Frances Woolfolk Wallace" | name == "Mary Jeffreys Bethell" | name == "Julia Johnson Fisher")
+
+diary.dtm <- diary_old %>% 
+  count(file_name, word) %>% 
+  cast_dtm(file_name, word, n)
+
+diary.lda <- LDA(diary.dtm, k = 9, control = list(seed = 12345),  method = "Gibbs", alpha = .9)
+diary.lda
+
+diary.topics <- tidy(diary.lda, matrix = "beta")
+head(diary.topics)
+
+diary.top.terms <- diary.topics %>%
+  arrange(desc(beta)) %>% 
+  group_by(topic) %>% slice(1:8)
+
+
+diary.top.terms %>%
+  mutate(term = reorder_within(term, beta, topic)) %>%
+  ggplot(aes(beta, term, fill = factor(topic))) +
+  geom_col(show.legend = FALSE) +
+  facet_wrap(~ topic, scales = "free") +
+  scale_y_reordered()
+```
```

Comment:
this is another code block where I would argue we don't need to see all of this in the text of your paper. If you split this into two blocks - one for lines 155-169 (echo false) and one for lines 172-177, you'd only see the resulting visualization which would allow the reader to focus on the argument and not the nuances of the code.